aws/structure.go

```diff
@@ -1477,6 +1477,7 @@ func flattenConfigSnapshotDeliveryProperties(p *configservice.ConfigSnapshotDeli
 	return []map[string]interface{}{m}
 }
 
+// TODO This should really be named 'pointersMapToStringMap'?
```

Comment:
Intentional to leave these comments? (seems unrelated to this PR?)

Reply:
Indeed, it must have just been left over from when I was working on the code.
Once some more of the prerequisite PRs for "lower" API Gateway v2 APIs have been merged I will rebase this one and remove the comments (and maybe revisit `diffStringMaps` - the intervening work on `keyvaluetags` may have some code I can reuse).